exo-deploy/terraform/aws/aws.tf

```diff
@@ -0,0 +1,60 @@
+data "aws_region" "current" {
+  current = true
+}
+
+data "aws_availability_zones" "available" {}
+
+module "internal_dns" {
+  source = "./internal-dns"
+
+  name    = "${var.name}.local"
```

Comment:
I think namespacing with the environment makes sense because for external DNS you have to do it anyways, so it makes sense to keep things consistent.

At least we should set the comment to something more meaningful than `""` so that they can be differentiated in the UI (currently if you have 5 env, you have 5 identically named zones)